portable-screencap

```diff
@@ -50,15 +51,15 @@ please install a supported one:
 }
 
 all_screencap() {
-  case `uname` in
-  Darwin) darwin_screencap $1 ;;
-  Linux) linux_screencap $1 ;;
-  *) die "screencap not implemented for "`uname` ;;
+  case $(uname) in
+  Darwin) darwin_screencap "$1" ;;
+  Linux) linux_screencap "$1" ;;
+  *) die "screencap not implemented for $(uname)" ;;
   esac
 }
 
 date=$(date +"%Y-%m-%dT%H:%M:%SZ")
-fpath=$(mktemp /tmp/screencap.$date.XXXXXX.png)
-all_screencap $fpath
-cat "$fpath"
-rm "$fpath" || true
+fpath=$(mktemp "/tmp/screencap.$date.XXXXXX.png")
+all_screencap "$fpath" || die "capture failed"
+echo "Capture is in '$fpath'"
```

Comment:
i want this script to output the file to stdout. storing in files makes other things (like chaining operations) harder. `portable-screencap >filename` is easy enough.